static/css/shared/_tiles.scss

```diff
@@ -629,4 +629,63 @@ $box-shadow-8dp: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
   border: var(--border-primary);
   border-radius: var(--border-radius-primary);
   padding: 32px;
- }
\ No newline at end of file
+ }
+
+ /**
+  * Answer table tile
+  */
+
+.answer-table-tile {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  margin-top: 0;
+  
+  .answer-table-title {
+    font-size: 32px;
+  }
+
+  .answer-table {
+    font-size: 24px;
+    font-weight: 400;
+    border: 1px solid #DEE2E6;
+    border-radius: 3px;
+    margin-top: 48px;
```

Comment:
supernit: please alphabetize the css fields

Reply:
done